tests/kernel/virtio_console.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Log into and run a script on a virtio serial terminal
+# Maintainer: Richard Palethorpe <rpalethorpe@suse.com>
+
+use 5.018;
```

Comment:
Perl best practices recommends it because it implies strict, enables any non experimental features up to 5.018 and reminds me what version of Perl we have on our server.